sdk/python/lib/pulumi/provider/experimental/analyzer.py

```diff
@@ -480,9 +506,15 @@ def get_docstring(self, typ: type, name: str) -> Optional[str]:
                         try:
                             node = next(it)
                             # Look for an assignment with a type annotation
-                            if isinstance(node, ast.AnnAssign):
-                                if isinstance(node.target, ast.Name):
-                                    target = node.target.id
+                            if isinstance(node, ast.AnnAssign) or isinstance(
+                                node, ast.Assign
+                            ):
+                                if isinstance(node, ast.AnnAssign):
+                                    target_node: ast.expr = node.target
+                                else:
+                                    target_node = node.targets[0]
+                                if isinstance(target_node, ast.Name):
+                                    target = target_node.id
```

Comment:
Worth bulking out the comment on this block? It's not super clear to me what's going on/why we've changed it

Reply:
Definitely, I added a comment in https://github.com/pulumi/pulumi/pull/19072/commits/04b4946a7929c8193e4895386c9944eca29d6863